server.cjs

```diff
@@ -76,7 +76,7 @@
   });
 
   function checkGzipAndNext(req, res, next) {
-    const reqUrl = url.parse(req.url, true);
+    const reqUrl = url.URL(req.url, true);
```

Comment:
This is causing some internal server errors when running Sandcastles such as [3D Tiles Adjust Height](http://localhost:8080/Apps/Sandcastle/index.html?src=3D%20Tiles%20Adjust%20Height.html).

You'll need to add the `new` keyword and also the second parameter is not a boolean, [it's a string which specifies the base URL](https://nodejs.org/api/url.html#class-url).

Also, for the to be a bit cleaner, you can use a destructor to only import the parts of `url` that we need

````js
const { format, URL } = require("url");
````